geode-docs/developing/function_exec/how_function_execution_works.html.md.erb

```diff
@@ -39,9 +39,18 @@ See the `org.apache.geode.cache.execute.FunctionService` Java API documentation
 
 The following things occur when executing a function:
 
-1.  When you call the `execute` method on the `Execution` object, <%=vars.product_name%> invokes the function on all members where it needs to run. The locations are determined by the `FunctionService` `on*` method calls, region configuration, and any filters.
-2.  If the function has results, they are returned to the `addResult` method call in a `ResultCollector` object.
-3.  The originating member collects results using `ResultCollector.getResult`.
+1.  For security-enabled clusters, prior to executing the function,
+a check is made to see that that caller is authorized to execute
+the function on the region.
```

Comment:
Function execution isn't always associated with a particular region.  (It will be if the function is called through `FunctionService.onRegion`, but not if the function is called through `FunctionService.onMembers`.)  So, we may want to simply say: 

```
For security-enabled clusters, prior to executing the function,
 a check is made to see that that caller is authorized to execute
 the function.
```